### PR TITLE
improve (CircleCI): Add integration tests

### PR DESCRIPTION
**Pull request by evaldofelipe**
**Motivation**

Add new integration tests on CI pipeline

**Testing**

Check a box to describe how you tested these changes and list the steps for reviewers to test.

- [ ]  Ran end-to-end test, running the code as in production
- [ ]  New unit tests created
- [ ]  Existing tests adequate, no new tests required
- [ ]  All existing tests pass
- [ ]  Untested

### CODE REVIEW

File: packages/core/migrations/2_deploy_finder.js [hunk 1]
```diff
@@ -5,4 +5,5 @@ module.exports = async function (deployer, network, accounts) {
   const keys = getKeysForNetwork(network, accounts);
 
   await deploy(deployer, network, Finder, { from: keys.deployer });
+  // test comment
```
Comment (chrismaree) [inline packages/core/migrations/2_deploy_finder.js:8]: should this still be here? I'm guessing you added it to validate that CI steps ran correctly.

File: scripts/ovm/run-optimism.sh [hunk 2]
```diff
@@ -1,9 +1,13 @@
 #!/bin/bash
 
+# download pre-builted images
+repo="ethereumoptimism"
+for image in {builder,l2geth,deployer};do docker pull $repo/$image; done
+
 # build in 2 steps
 function build_images() {
-  docker-compose build --parallel -- builder l2geth l1_chain
-  docker-compose build --parallel -- deployer dtl batch_submitter relayer
```
Comment (chrismaree) [inline scripts/ovm/run-optimism.sh:6]: so we dont need to build these ones as they are pulled from `ethereumoptimism`, right?

Comment (evaldofelipe) [reply scripts/ovm/run-optimism.sh:6]: yeah, but we are able to pull just 3 images, and the other 4 images we need build it.

Comment (chrismaree) [reply scripts/ovm/run-optimism.sh:6]: I see. nice optimisation

**Review by chrismaree — APPROVED** [review]
This is awesome! thanks for adding the config.

**Pull request merged**